src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.ts

```diff
@@ -28,16 +28,20 @@ export class DotEditLayoutService {
         const grid: DotLayoutGridBox[] = [];
 
         dotLayoutBody.rows.forEach((row, rowIndex) => {
-            row.columns.forEach((column) => {
+            row.columns.forEach(column => {
                 grid.push({
-                    containers: column.containers.map((dotPageContainer: DotPageContainer) => {
-                        return {
-                            container: this.templateContainersCacheService.get(
-                                dotPageContainer.identifier
-                            ),
-                            uuid: dotPageContainer.uuid
-                        };
-                    }),
+                    containers: column.containers
+                        .filter(dotPageContainer =>
```

Comment:
What we do here and in the sidebar is the exact same thing, can we extract that functionality into a method and stay DRY?